services/wikiService.js

```diff
@@ -0,0 +1,25 @@
+var request = require('request');
+
+module.exports = function (cb,username,input) {
+    'use strict';
+    let apiOptions = {
+        url :`https://en.wikipedia.org/w/api.php?format=json&action=query&generator=search&gsrnamespace=0&gsrlimit=4&prop=pageimages|extracts&pilimit=max&exintro&explaintext&exsentences=1&exlimit=max&gsrsearch=${input}`,
```

Comment:
@yashmehrotra I'm implementing a better method as a part of my own PR.
@priyansh1161 For now, please declare a const and use it for the url.